vendor/assets/stylesheets/dvl/core/includes.scss

```diff
@@ -35,13 +35,15 @@ $radius: 3px !default;
 
 // Font weights
 $weightLight: 300 !default;
-$weightNormal: 500 !default;
+$weightNormal: 400 !default;
```

Comment:
Open Sans doesn't have a `500` weight, only Aktiv Grotesk, so this is a mistake.